src/main/java/io/lakefs/iceberg/LakeFSReporter.java

```diff
@@ -0,0 +1,81 @@
+package io.lakefs.iceberg;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.hadoop.conf.Configuration;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LakeFSReporter {
+    private URI lakeFSEndpoint;
+    private Configuration hadoopConfig;
+    private HttpClient httpClient;
+    private final String reportClient = "iceberg-catalog/" + getClass().getPackage().getImplementationVersion();
+
+    private LakeFSReporter(){}
+    public LakeFSReporter(Configuration hadoopConfig) {
+        String lakeFSServerURL = hadoopConfig.get("fs.s3a.endpoint");
+        if (lakeFSServerURL.endsWith("/")) {
+            lakeFSServerURL = lakeFSServerURL.substring(lakeFSServerURL.length()-1);
+        }
+        lakeFSServerURL += "/api/v1/statistics";
+        this.lakeFSEndpoint = URI.create(lakeFSServerURL);
```

Comment:
```suggestion
       this.lakeFSEndpoint = URI.create(StringUtils.stripEnd(lakeFSServerURL, "/") + "/api/v1/statistics");
```
(can also split to 2 lines)

Reply:
done